app/post.go

```diff
@@ -881,21 +862,19 @@ func makeOpenGraphURLsAbsolute(og *opengraph.OpenGraph, requestURL string) {
 
 func (a *App) DoPostAction(postId, actionId, userId, selectedOption string) *model.AppError {
 	pchan := a.Srv.Store.Post().GetSingle(postId)
+	cchan := a.Srv.Store.Channel().GetForPost(postId)
```

Comment:
Is there a potential performance improvement if you start the code with

```
pchan := a.Srv.Store.Post().GetSingle(postId)
cchan := a.Srv.Store.Channel().GetForPost(postId)
```
and get the results later?